test/e2e/gameserver_test.go

```diff
@@ -852,6 +848,28 @@ func TestGameServerPassthroughPort(t *testing.T) {
 	assert.Equal(t, "ACK: Hello World !\n", reply)
 }
 
+func TestGameServerPortPolicyNone(t *testing.T) {
+	framework.SkipOnCloudProduct(t, "gke-autopilot", "does not support None PortPolicy")
```

Comment:
That's not true, or implemented, AFAICT. :) (It looks like Autopilot should support this just fine, but separately, if you intended it not to be the case, you would need to modify the validation webhook.)

Reply:
Should I also change the [error message](https://github.com/daniellee/agones/blob/gameservers-pod-ip/pkg/cloudproduct/gke/gke.go#L42) "portPolicy must be Dynamic on GKE Autopilot" to something like "portPolicy must be Dynamic or None on GKE Autopilot"? Does that make sense?

Reply:
That's a good idea, yeah!